docs/examples/redshift.rst

```diff
@@ -0,0 +1,68 @@
+Redshift Estimation
+===================
+
+The project described on this page can be found in the `deepforge examples repository <https://github.com/deepforge-dev/examples>`_ on GitHub under the name **redshift-example.webgmex**
```

Comment:
Once the examples have been added to the examples repo, we could link directly to them from here.

Reply:
We can also probably just say "examples repo". Otherwise, we should probably capitalize "deepforge" to "DeepForge".

Reply:
Links changed to expected subfolder location in examples repo. Agreed on removing unnecessary "DeepForge" modifier. Both changes were made to both this file and rs-tutorial.rst